src/sentry/integrations/slack/analytics.py

```diff
@@ -21,7 +21,7 @@ class SlackIntegrationNotificationSent(analytics.Event):  # type: ignore
     type = "integrations.slack.notification_sent"
 
     attributes = (
-        analytics.Attribute("organization_id"),
+        analytics.Attribute("organization_id", required=False),
```

Comment:
@mgaeta can you please add the `integrations.slack.notification_sent` to https://github.com/getsentry/etl/blob/master/etl/operators/analytics_events_schema.py?